quickstep/src/com/android/quickstep/QuickstepProcessInitializer.java

```diff
@@ -65,7 +65,7 @@ protected void init(Context context) {
 
                 // Elevate GPU priority for Quickstep and Remote animations.
                 try {
-                        if (!Utilities.ATLEAST_Q) return;
+                        if (!Utilities.ATLEAST_P) return;
```

Comment:
What's the story here?

Reply:
Android 8.x don't have `EGL_CONTEXT_PRIORITY_HIGH_IMG` so it will crash anyway with (if I remember) no such method.